test-data/unit/deps.test

```diff
@@ -1346,3 +1346,34 @@ class D(C):
 <m.C.mm> -> m.D.mm
 <m.C> -> m, m.C, m.D
 <m.D> -> m.D
+
+[case testDataclassDeps]
+# flags: --python-version 3.7
+from dataclasses import dataclass
+
+Z = int
+
+@dataclass
+class A:
+    x: Z
+
+@dataclass
+class B(A):
+    y: int
+[builtins fixtures/list.pyi]
+
+[out]
+<m.A.(abstract)> -> <m.B.__init__>, m
+<m.A.__eq__> -> <m.B.__eq__>
+<m.A.__init__> -> <m.B.__init__>, m.B.__init__
+<m.A.__ne__> -> <m.B.__ne__>
+<m.A.__new__> -> <m.B.__new__>
+<m.A.x> -> <m.A[wildcard]>, <m.B.x>
+<m.A.y> -> <m.B.y>
+<m.A> -> m, m.A, m.B
+<m.A[wildcard]> -> m
+<m.B.x> -> <m.B[wildcard]>
```

Comment:
This looks unnecessary, `calculate_active_triggers` should activate the wildcard if `astdiff` returns `m.B.x`.